galsim/scene.py

```diff
@@ -401,7 +549,7 @@ def _makeParametric(self, indices, chromatic, sersic_prec, gsparams):
 
         gal_list = []
         for index in indices:
-            record = self.param_cat[self.orig_index[index]]
+            record = self.getParametricRecord(index)
```

Comment:
Looks like I've been commenting on a fair amount of old code that just happened to show up in the imperfectly processed github diff.  Hope that's not too annoying.  I actually have a comment for some old code on line 751 in `getParametricRecord`, which isn't imperfectly diffed, so github won't let me comment on that line, so I'll place it here instead.  I was happy to see a generator expression:

```
record_dict = dict( ((k,record[k]) for k in record.dtype.names) )
```

I just wanted to point out that a generator expression passed as a single argument to a function doesn't need its own parens.  I.e., the following works and, at least to me, looks a bit cleaner:

```
record_dict = dict((k,record[k]) for k in record.dtype.names)
```